gradle/docker.gradle

```diff
@@ -42,14 +58,42 @@ if(project.hasProperty('dockerBuildArgs')) {
     }
 }
 
+task templateDockerfile (type: Copy) {
+  def directory = project.buildscript.sourceFile.getParentFile()
+  from directory
+  into directory
+  include 'Dockerfile.in'
```

Comment:
I'm trying to grok this new gradle process.
Where is Dockerfile.in coming from?

From I grok the templateDockerfile or rmDockerfile is need it since you are passing `-f Dockerfile.s390x` when your building the specia image for the s390x image

Reply:
There are two options for Dockerfile.  'Dockerfile.s390x' if it exists will completely override 'Dockerfile'.  Option 2 is 'Dockerfile.in' would be run through the groovy template engine.  It's intended for small differences in the Dockerfile (e.g. Debian releases).  I'm not sure Option 2 is even in use here, and so perhaps I should just rip it out for the sake of readability/clarity.  I had left it in in case it was needed later.  At any rate, I need better comment documentation apparently.